crates/subspace-networking/src/create.rs

```diff
@@ -65,7 +65,8 @@ const YAMUX_MAX_STREAMS: usize = 256;
 ///
 /// We restrict this so we don't exceed number of incoming streams for single peer, but this value
 /// will be boosted depending on number of connected peers.
-const KADEMLIA_BASE_CONCURRENT_TASKS: usize = 30;
+const KADEMLIA_BASE_CONCURRENT_TASKS: NonZeroUsize =
+    NonZeroUsize::new(30).expect("KADEMLIA_BASE_CONCURRENT_TASKS is 0");
```

Comment:
It is not zero though. `.expect()` is not something we expect application to print as an error, this is something that never happens and message must contain a proof. In this case:
```suggestion
    NonZeroUsize::new(30).expect("Not zero; qed");
```

Reply:
Hm, in this case it failed to create NonZeroUsize as the values passed to new() was 0. May be I don't understand your point?

Reply:
It didn't, that is the point. I already elaborated on this in https://subspacenetwork.slack.com/archives/C03R16SFFJM/p1668314727705509 as mentioned before